instrumentation/netty-reactor-0.7.0/src/main/java/reactor/ipc/netty/http/server/HttpServerHandler_Instrumentation.java

```diff
@@ -11,13 +11,23 @@
 import com.newrelic.api.agent.weaver.MatchType;
 import com.newrelic.api.agent.weaver.Weave;
 import com.newrelic.api.agent.weaver.Weaver;
+import io.netty.TokenLinkingSubscriber;
+import io.netty.channel.ChannelHandlerContext;
 import io.netty.channel.ChannelHandlerContext_Instrumentation;
 import io.netty.handler.codec.http.HttpRequest;
+import reactor.core.publisher.Hooks;
+import reactor.core.publisher.Hooks_Instrumentation;
+
+import static io.netty.TokenLinkingSubscriber.tokenLift;
 
 @Weave(type = MatchType.BaseClass, originalName = "reactor.ipc.netty.http.server.HttpServerHandler")
 class HttpServerHandler_Instrumentation {
 
     public void channelRead(ChannelHandlerContext_Instrumentation ctx, Object msg) {
+        //TODO: Find a better place for this or add static initializer support for the weaver
```

Comment:
Is this going to be addressed as part of this PR? Or should we remove the comment and file a new issue to do this work in the future?